docs/resources/application_key.md

```diff
@@ -26,6 +26,10 @@ resource "datadog_application_key" "foo" {
 
 - **name** (String) Name for Application Key.
 
+### Optional
+
+- **service_account** (String) ID of a service account that will own the Application Key
```

Comment:
```suggestion
- **service_account** (String) ID of a service account that owns the Application Key.
```

Reply:
Thanks, fixed that..

Reply:
@jake-scott I don't think you added this suggestion in, try one more time.

Reply:
ah sorry, forgot the 'make docs', should be good now..